ERCS/erc_draft_metadata_management_for_nfts.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Metadata Management for NFTs
+description: This ERC proposes a standard for managing metadata of NFTs that allows owners and authorized addresses to update the metadata of their tokens.
+author: Alexey Kureev (@Kureev) <a.g.kureev@gmail.com>
+discussions-to:
```

Comment:
Please add a topic to Eth Magicians forum with a link to the PR and then add link here.